socialhome/streams/app/components/StreamElement.vue

```diff
@@ -6,126 +6,66 @@
             v-bind="author"
             :is-user-author="isUserAuthor"
             :is-user-local="isUserLocal"
-            :current-browsing-profile-id="currentBrowsingProfileId"
-            :is-user-authenticated="isUserAuthenticated"
         >
         </author-bar>
-        <div class="grid-item-bar d-flex justify-content-start">
+        <reactions-bar
```

Comment:
How come we're calling a component and then still having content here? What does it mean?

Reply:
Content inserted inside the component tag can be rendered by the component with [the `<slot />` tag](https://vuejs.org/v2/guide/components.html#Single-Slot).